src/main/flight/pid.c

```diff
@@ -445,19 +446,62 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #ifdef USE_GPS_RESCUE
     angle += gpsRescueAngle[axis] / 100; // ANGLE IS IN CENTIDEGREES
 #endif
-    angle = constrainf(angle, -levelAngleLimit, levelAngleLimit);
     const float errorAngle = angle - ((attitude.raw[axis] - angleTrim->raw[axis]) / 10.0f);
+    const float levelPGainBoost = (1.0f + (fabsf(errorAngle) / levelAngleLimit));
```

Comment:
I think we should re-name this to `levelErrorGainBoost`

Reply:
done!